client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -116,7 +126,10 @@ HistoryViewerVersionDetail.propTypes = {
   recordId: PropTypes.number.isRequired,
   schemaUrl: PropTypes.string.isRequired,
   ToolbarComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
-  version: versionType.isRequired,
+  version: versionType,
+  compareMode: PropTypes.bool,
+  compareFrom: versionType,
+  compareTo: versionType,
```

Comment:
I'm +1 for this. I feel like it's getting a bit convoluted. I would call the method above just `isCompareMode` rather than `getIsCompareMode` though 🙂